ee-9/feature-pack/pom.xml

```diff
@@ -481,6 +481,9 @@
                                      functionality (e.g. consoles or similar) that we do not support. -->
                                 <exclude>com.h2database:h2\z</exclude>
                                 <exclude>org.apache.thrift:libthrift\z</exclude>
+                                <!-- Only contains hard-coded strings that are either not used or both the javax and
+                                jakarta names are used -->
+                                <exclude>org.apache.logging.log4j:log4j-api\z</exclude>
```

Comment:
@jamezp Please move this down. These are not strictly alphabetical; they are grouped by the explanatory comment so this PR now separates jasypt from its explanation.

Reply:
Done. Moved to the bottom of the exclusions as it didn't seem to fit in the other categories.